spec/index.html

```diff
@@ -5462,29 +5462,33 @@ <h5>RDFterm-equal</h5>
-              arguments are both literal but are not the same RDF term <sup><a href=
-                                                                               "#func-RDFterm-equal-foot1" class="footnote">*</a></sup>; returns FALSE otherwise.
-              <code>term1</code> and <code>term2</code> are the same if any of the following is
+            <p>Returns TRUE if <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms as defined below;
+              produces a type error if the arguments are both literals with the same datatype that is unsupported by the implementation but are not the same RDF term <sup><a href="#func-RDFterm-equal-foot1" class="footnote">*</a></sup>;
+              returns FALSE otherwise.</p>
+            <p>
+              <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms if any of the following is
               true:</p>
             <ul>
               <li>
-                <span class="name">term1</span> and <span class="name">term2</span> are equivalent
+                <span class="name">term1</span> and <span class="name">term2</span> are equal
                 <span class="IRI type">IRIs</span> as defined in <a data-cite=
-                                                                    "RDF12-CONCEPTS#section-Graph-URIref">6.4 RDF URI References</a> of
+                                                                    "RDF12-CONCEPTS#dfn-iri-equality">IRI quality</a> of
                 [[RDF12-CONCEPTS]].
               </li>
               <li>
-                <span class="name">term1</span> and <span class="name">term2</span> are equivalent
+                <span class="name">term1</span> and <span class="name">term2</span> are equal
                 <span class="literal type">literals</span> as defined in <a data-cite=
-                                                                            "RDF12-CONCEPTS#section-Literal-Equality">6.5.1 Literal Equality</a> of
+                                                                            "RDF12-CONCEPTS#dfn-term-equal">Literal term equality</a> of
                 [[RDF12-CONCEPTS]].
               </li>
+              <li>
+                <span class="name">term1</span> and <span class="name">term2</span> are
+                <span class="literal type">literals</span> with equivalent values according to their datatypes.
+              </li>
               <li>
                 <span class="name">term1</span> and <span class="name">term2</span> are the same
                 <span class="bnode type">blank node</span> as described in <a data-cite=
-                                                                              "RDF12-CONCEPTS#section-blank-nodes">6.6 Blank Nodes</a> of [[RDF12-CONCEPTS]].
+                                                                              "RDF12-CONCEPTS#section-blank-nodes">Blank Nodes</a> of [[RDF12-CONCEPTS]].
```

Comment:
Currently, the section that you reference here does not talk about "same" blank nodes. However, I think it is okay to remove the reference; i.e., just end the sentence after: "[...] same blank node."

Reply:
Ok, I agree. Removed in the latest commit.